### ISSUE TITLE
Having trouble with --real-time-html refreshing

### ISSUE THREAD

**Issue by MrDotOne** [allinurl/goaccess#2446]
CentOS 7.9.2009 with a built from source version of 1.7. It runs fine, however the following command line 

 goaccess /var/log/httpd/ssl_access_log -o /var/www/html/vision/web/test/report.html --log-format=COMMON --real-time-html

runs fine, but never seems to refresh. I have tested from various sources that 7890 is indeed open (using netcat). And i am really at a loss. We would love to have real-time or even semi real-time data for our web developers to access so they know what they need to work on.

Any thoughts?

**Comment by allinurl** [comment]
Could you please post what's the output from your browser's console? also, is the dot next to the gears icon (nav bar) green?

**Comment by MrDotOne** [comment]
I am not sure what you mean by 'what's the output from your browser's console' and i do not see a dot green or otherwise on the report.html page

**Comment by allinurl** [comment]
If you are using Google Chrome, please press Ctrl+Shift+j and take a screenshot of that and post it here. If there's no green dot, it means the browser is not connecting to the websocket server.

**Comment by windhamdavid** [comment]
@allinurl . I just wanted to leave a note here that I'm unable to get the sockets working in Safari. It's fine in Chrome and Firefox. I'm using wss with ssl-cert in the goaccess.conf.  Otherwise, big Thank You 👊🏼 for you work on this.

**Comment by allinurl** [comment]
@windhamdavid Thanks for sharing this. Are you getting any errors on Safari's console when run from the console?

**Comment by windhamdavid** [comment]
@allinurl - I retested this and I'm now unable to duplicate the error. I tried removing my custom config as well. Unfortunately, it looks like the error was on my end. I don't remember seeing any console errors in Safari which is why I reported it here. Perhaps it was related to local storage as I had previously loaded up serveral other versions before testing other browsers. Searching the :issues here shows how much support you've had to give with websockets.  I cron weekly goaccess live dashboards for Nginx/Apache with very little memory footprint and it saves me the time and effort of sorting through the logs. Thank you 👍🏼
